magento/magento2
# Added progress bar to multiple shipping address checkout

### Description (*)


Progress bar was missing on multiple shipping address checkout so added it.

**After:**

![after-progress-bar](https://user-images.githubusercontent.com/5336201/86944163-f9334a80-c164-11ea-9d76-6ad1c8ba5397.png)

**Before:**

![before-progress-bar](https://user-images.githubusercontent.com/5336201/86944157-f6d0f080-c164-11ea-86ff-351d2d299af6.png)

### Related Pull Requests


None

### Fixed Issues (if relevant)


None

### Manual testing scenarios (*)

1. Add any two products to the cart
2. Use "Check Out with Multiple Addresses" option from cart

### Questions or comments


None

### Contribution checklist (*)
